torch_xla/csrc/ir_dump_util.h

```diff
@@ -25,7 +25,8 @@ class DumpUtil {
       absl::Span<const torch::lazy::Node* const> roots);
 
   static std::string ToHlo(c10::ArrayRef<torch::lazy::Value> values,
-                           const torch::lazy::BackendDevice& device);
+                           const torch::lazy::BackendDevice& device,
+                           bool to_stable_hlo = false);
```

Comment:
Nit: the canonical spelling for StableHLO in code is "stablehlo" or "Stablehlo" (without underscores or capitalization in the middle of the word).